pkg/master/controller.go

```diff
@@ -41,8 +40,6 @@ import (
 type Controller struct {
 	NamespaceRegistry namespace.Registry
 	ServiceRegistry   service.Registry
-	// TODO: MasterCount is yucky
-	MasterCount int
```

Comment:
Do not remove this until there's a replacement. The other parts of this PR are not a replacement as far as I can see.